geoportal/c2cgeoportal_geoportal/lib/__init__.py

```diff
@@ -396,3 +397,14 @@ def add(self, config, name, spec, **extra):
             extra["pregenerator"] = C2CPregenerator(subdomain=True, version=False)
         return super(MultiDomainStaticURLInfo, self) \
             .add(config, name, spec, **extra)
+
+
+_formatter = Formatter()
+
+
+def confd_env(val):
```

Comment:
I not sure to understand, is it possible to add a docstring, an example ? 
=> Mako filter that allow to use environnement variables in mako templates. 
${"{host}:{port}" | confd_env} => {{getenv "host"}}:{{getenv "port"}}

Reply:
${ host | confd_env}:${ port | confd_env} => {{getenv "host"}}:{{getenv "port"}}
I will add a comment :-)